README.md

```diff
@@ -498,13 +498,13 @@ The [Google Places Search API](https://developers.google.com/places/web-service/
 #### LocationIQ (`:location_iq`)
 
 * **API key**: required
-* **Quota**: 6 request/second (30k req/day), then ability to purchase more
+* **Quota**: 60 requests/minute (2 req/day, 10k req/day), then ability to purchase more
```

Comment:
What's the 2 req/day for? Documentation seems to inidicate just 10K req/day.

Reply:
We clarified the rate limits [under FAQs](https://locationiq.org/#faqs)